### ISSUE TITLE
Debug problem

### ISSUE THREAD

**Issue by BertoMe** [Proxicon/sim7600nF#2]
Hi, I have been trying your code but I have two problems, one when compiling although I have solved it by omitting the lines that produce the error, and another problem is when debugging the code:

![imagen](https://user-images.githubusercontent.com/92958492/235329805-0ede603b-6419-4c3a-91b6-895fe6196dc9.png)

**Comment by Proxicon** [comment]
Hello, that is because of the tinygps module that in effect requires older
versions of the nanoframework binaries.

I have added the tinygps module into the project since it's open source and
available on GitHub and updated their references that makes this issue go
away. I'll upload it later on the Dev branch.

To note as well I'm uncertain if I'll continue to use the tinygps module
for this project since for the sim7600 GPS data the format is different
than what the module supports.

**Comment by Proxicon** [comment]
@BertoMe - Ok I see what I did, pull this repo and update all the nanoframework references to the latest: https://github.com/mboud/TinyGPSPlusNF

I Moved the compiled .dll to my local project folder and added a reference to the .dll on the local filesystem.
Initially I did keep the TinyGPS module inside of this one but it felt a little bloated.

Let me know if this solves your issue.

![image](https://user-images.githubusercontent.com/14370617/235473509-8d586b72-0afd-46f7-924d-6a621f0e4175.png)

As mentioned before I dont think ill continue to use TinyGPS in this module since the Sim7600 GPS data mismatched the modules data. Need more playtime to validate that soon. I might just end up posting the data to a DB like my other pinned project allowing for more options with the data in a full asp.net app.

This project is in a state where I can init the sim7600 and obtain a network connection + data connection.
My goals are to http post via rest to my own hosted API which at the minute requires some fine tuning of memory usage on the esp32 device. I got the HTTP post working for retrieving a rest API authentication token whereby I'm sort of ready to start porting GPS data via HTTP Rest calls but my esp device ran out of memory after API token retrieval.

So im getting ready to trim unused code and do a little object cleanup - see the latest commit in dev.

**Comment by BertoMe** [comment]
I'll tell you what I've done now, I've downloaded your project again, compiled it and sent it to Esp32:

![imagen](https://user-images.githubusercontent.com/92958492/235797433-191ad004-9475-4986-b18e-e459b54a2588.png)

But it keeps giving me errors:

![imagen](https://user-images.githubusercontent.com/92958492/235797516-1f3fb5f2-bd10-41e0-9917-6b9410a43b38.png)

If you want, I can share a Anydesk connection.

**Edit: Now work, but only if remove TinyGPSPlusNF from sim7600x.cs**

**Comment by Proxicon** [comment]
Yes as mentioned you need to build [TinyGPSPlusNF](https://github.com/mboud/TinyGPSPlusNF) yourself and update the nF assemblies it references before you do.  Unfortunately I burned some time as well before I realized that was the problem.

I'm not using TinyGPSPlusNF in the project yet and have removed it on [pull request 4](https://github.com/Proxicon/sim7600nF/pull/4) from the dev & master branches & will consider adding it back after I have finished my http rest posting sequence which is almost there.
